Homepage content

msecret opened the pull request:
This is the full homepage with the multiple sections, only lacking the footer. Tracks https://github.com/18F/cg-style/pull/10 in cg-style.

Commit:
Cleanup

There shouldn't be two separate places where assets like js, css, fonts
and images are kept. I'm keeping them all in an assets dir that gets
copied to the publish _site dir for better organizaion.

Commit:
Added all cg-style assets

Moves all the fonts and img assets for cg style into the static dir.
Also moves the cg-style css file into the assets dir for import
directly into html. This was done because importing into a sass file
isn't working because of some jekyll bullshit.

Commit:
Header from cg-style

Tracking "override-disclaimer" branch on cg-style.

Commit:
Billboard and sections

Tracking cg-style#bilboard.

Commit:
More parts of the sections

Stacking back and forth light and dark.

Commit:
Rest of content on homepage

Rest of content from Jame's content document.

Tracks cg-style#ups

Commit:
Finshed testimonials content section

Commit:
Last section on homepage.

Commit:
Basic contact form

Commit:
Contact for styling

Using an adjacent sibling selector class for when there should be
separation. Works the same as if its a p tag.

Had to do some inline styling, wasn't sure of best way.

Commit:
Footer

Commit:
Mobile

Tracks cg-style#mobile_and_all.

jcscottiii commented on pages/home.md:
```diff
@@ -3,5 +3,5 @@ layout: home
 hero-image: /assets/img/clouds.jpg
 hero-text: "The Government Innovation Platform"
-hero-subtext: "Free your team to focus on creating quality services securely hosted in the cloud"
+hero-subtext: "We built it. We manage it. We keep improving it. Want to use it?"
```
Was this in the design docs?? maybe I just can't find it lol.

dlapiduz replied:
This feels a bit pushy... @mogul did you write it?

msecret replied:
It was in James Hupp's document: https://docs.google.com/document/d/1EJEToDGuloXp1GMTqpgdCxLQwF2Z5bopdwGm-UUWMOQ/edit#. It was one of a few options, please pick whichever one you like.

mogul replied:
Hey @dlapiduz, why would you think pushy text was me?! _hurt_

jcscottiii commented on js/happyfoxchat.js:
```diff
@@ -1,18 +0,0 @@
-      HOST_URL: "https://www.happyfoxchat.com",
-      ASSETS_URL: "https://d1l7z5ofrj6ab8.cloudfront.net/visitor"
-  };
-
```
might have to talk to @mogul about removing the chat from the landing page.

msecret replied:
I can add it back

mogul replied:
I'm okay with removing it... It's clearly not getting used much. I care more about it on the deck.

jcscottiii commented on js/18fanalytics.js:
```diff
@@ -1,10 +0,0 @@
-(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
-
```
need to keep this

msecret replied:
Is this file actually being imported anywhere? I can't find where it is.

jcscottiii replied:
@geramirez any word on this?

geramirez replied:
as long as we have the same analytics snippet somewhere it should be ok.

jcscottiii replied:
ahh as long as the page loads it. gotcha. thanks @geramirez !! no import needed

msecret replied:
Yea and there is an analytics snippet already existing in javascript.html

jcscottiii commented:
Looks good @msecret. Had a few suggestions. and additionally some of the final css might change depending on my suggestions here: https://github.com/18F/cg-style/pull/13. Also, lets talk about what you want to do with the jquery removal for the site that's causing the wercker failures.

jcscottiii committed:
Add styling dev instructions

Commit:
Fix JS, new display secondary text

Adds back in some JS and puts everything in one javascript include
rather then confusingly splitting it across two. Adds back in jquery
because I'm guessing it's actually required.

Uses a new tagline for the large display text due to discussion.

Commit:
Update push utility name

Tracking changes in cg-style#mobile_and_all

Commit:
Merge pull request #40 from 18F/add-styling-dev-instructions

Add styling dev instructions

Commit:
Fix some style issues with posts pages

Commit:
Merge remote-tracking branch 'origin/homepage_content' into homepage_content

Commit:
Remove hapy fox chat

jcscottiii commented:
Looks good @msecret! Thanks for cranking this out so quickly
